README.rst

```diff
@@ -0,0 +1,99 @@
+Before the start, make sure that you have an account on `PyPI`_
```

Comment:
"Before the start"?

Reply:
Changed to `Prior to the use of this cookiecutter, `